### ISSUE TITLE
Connection set up preferences page is empty

### ISSUE THREAD

**Issue by Tillerino** [nhat-phan/merge-request-integration#21]
I am running IntelliJ 2019.3.2 with merge request integration 2019.3.4. I am trying to set up the Gitlab connection, but the preferences page is completely empty:

![image](https://user-images.githubusercontent.com/1244517/73650661-f5197300-4682-11ea-889b-899ddb059b5f.png)

Help!

**Comment by Alzhurav** [comment]
I have the same issue (((

**Comment by dylanfontaine** [comment]
Seeing the same issue. Here's my ide info
```
PhpStorm 2019.3
Build #PS-193.5233.101, built on November 27, 2019
Runtime version: 11.0.4+10-b520.11 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 5.3.0-29-generic
GC: ParNew, ConcurrentMarkSweep
Memory: 1890M
Cores: 12
Registry: run.processes.with.pty=TRUE, ide.balloon.shadow.size=0
Non-Bundled Plugins: com.chrisrm.idea.MaterialThemeUI, com.kalessil.phpStorm.phpInspectionsEA, com.ppolivka.gitlabprojects, de.espend.idea.php.annotation, eu.osimowicz.plugins.intellij.CodeBlocksSorter, fr.adrienbrault.idea.symfony2plugin, net.ntworld.nhat-phan.merge-request-integration-ce, org.jetbrains.plugins.rest, ru.adelf.idea.dotenv
```

**Comment by evangalen** [comment]
Also seeing the same issue here on my IntelliJ IDEA Ultimate 2019.3.2

**Comment by edoreld** [comment]
Same here.

`IU-193.5662.53, JRE 11.0.5+10-b520.17x64 JetBrains s.r.o, OS Mac OS X(x86_64) v10.14.6, screens 1920x1200; Retina`

**Comment by nhat-phan** [comment]
Thank you for reporting the problem.

I found the root cause: if you are new users who don't have any connections, the gitlab configuration page is empty. I've released hot-fix version v2019.3.5. Please check it out.

I'm so sorry for the inconvenience.
